Scripts/Connector/Sample connector restore script.ps1

```diff
@@ -0,0 +1,40 @@
+# This script restores a connector that has been backed up.
+# To backup a Connector look at "Sample connector backup script.ps1"
+
+# RUN THIS SCRIPT IN AN ELEVATED POWERSHELL WINDOW
+
+# BEFORE RUNNING THESE COMMANDS PERFORM THE FOLLOWING STEPS:
+# 1) Copy the C:\ConnectorBackup folder (generated by the backup script) to the machine in which you will restore it
+# 2) Install the latest Connector from https://aka.ms/UPConnector (do no register the Connector)
+ 
+# Stop services
+net stop "Print Connector service"
+net stop PrintConnectorUpdaterSvc
+ 
+# Restore local printers
+c:\windows\system32\spool\tools\PrintBrm.exe -R -F C:\ConnectorBackup\printers.brm
+ 
+# Import UP certificates
+New-Item -Path "CERT:\LocalMachine\PrintProxyStore"
+$pwd = ConvertTo-SecureString -String "password" -AsPlainText -Force
+Get-ChildItem "C:\ConnectorBackup\Certs" | Foreach-Object { Import-PfxCertificate -Password $pwd -FilePath $_.FullName -CertStoreLocation Cert:\localmachine\PrintProxyStore -Exportable }
+ 
+# Import conector registries
+reg import C:\ConnectorBackup\Connector.reg
+ 
+# Import the Cloud Data registries
+Get-ChildItem "C:\ConnectorBackup\CloudData" | Foreach-Object { reg import $_.FullName }
+ 
+# Copy Connector config json
+copy C:\ConnectorBackup\config.json C:\windows\PrintConnectorSvc\
+ 
+# Copy Custom Mappings
+xcopy /S /Y c:\ConnectorBackup\PrintTicketMappings\ C:\ProgramData\Microsoft\UniversalPrintConnector\CustomPrintTicketMappings
+
+Write-Host "ENSURE THE CONNECTOR SERVICE HAS BEEN STOPPED IN THE OLD MACHINE BEFORE TURNING ON THE RESTORED CONNECTOR." -ForegroundColor Yellow
```

Comment:
nit: replace TURNING ON THE RESTORED CONNECTOR with PROCEEDING. the former implies they'll have to manually turn it on